framework/src/controller/action.ts

```diff
@@ -15,7 +15,7 @@
 import { strict as assert } from 'assert';
 import { actionWithModuleNameReg, moduleNameReg } from '../constants';
 import { RequestObject, VERSION } from './jsonrpc';
-import { ResponseObject, JSONRPCResult, ID, JSONRPCError } from './jsonrpc/types';
+import { ResponseObject, JSONRPCResult, ID, JSONRPCErrorObject } from './jsonrpc/types';
```

Comment:
maybe it missed last time due to auto import.. u could use `import { RequestObject, VERSION } from './jsonrpc';`?